diff --git a/CMakeLists.txt b/CMakeLists.txt
index 99bebaa98..dc2f9f74a 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -458,11 +458,13 @@ endif()
 # We use default paths provided from GNUInstallDirs. From CMake 3.14 these paths (CMAKE_INSTALL_...) will be used
 # automatically and can be removed from the following call. The same applies to many other calls to install(TARGETS...).
 # Note that GNUInstallDirs should still be included in the project root.
-install(TARGETS rmlui_core
-	EXPORT RmlUiTargets
-	${RMLUI_RUNTIME_DEPENDENCY_SET_ARG}
-	RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
-	LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
-	ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
-)
-install_target_pdb(rmlui_core)
+if(NOT "${CMAKE_HOST_SYSTEM_NAME}" STREQUAL "Android")
+	install(TARGETS rmlui_core
+		EXPORT RmlUiTargets
+		${RMLUI_RUNTIME_DEPENDENCY_SET_ARG}
+		RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
+		LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
+		ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
+	)
+	install_target_pdb(rmlui_core)
+endif()
